_sass/_post.scss

```diff
@@ -118,9 +118,24 @@
     font-size: 0.9rem;
 }
 
-
-
 .post-img {
     padding: 2%;
     max-width: 96%;
 }
+
+.fb-share-button {
+  text-align: center;
+  display: inline-block;
```

Comment:
This is duplicated code between `.twitter-share-button`, and maybe could be abstracted to the .social-buttons class!